ci/scripts/presto-integration.sh

```diff
@@ -30,9 +30,9 @@ rm -rf ./presto_cpp/main/lib
 cp -r ./BDTK/cpp/thirdparty/velox .
 mkdir -p ./presto_cpp/main/lib
 
-rm -rf BDTK/src
-mkdir -p BDTK/src
-cp -r ./BDTK/cpp/src/cider-velox BDTK/src
+sed -i 's/BDTK\/src/BDTK\/cpp\/src/g' presto_cpp/main/PrestoServer.cpp
```

Comment:
you can use s|xxx|xxx to skip escape \ to more readable in sed

Reply:
fix
sed -i 's|BDTK/src|BDTK/cpp/src|g' presto_cpp/main/PrestoServer.cpp
sed -i 's|BDTK/src|BDTK/cpp/src|g' presto_cpp/main/TaskResource.cpp